common/components/ConfirmationModal/components/Body/Body.scss

```diff
@@ -0,0 +1,104 @@
+@import 'common/sass/variables';
+
+.Body {
+  margin: auto;
+  max-width: 35rem;
+  width: 100%;
+  @media screen and (min-width: $screen-sm) {
+    min-width: 32rem;
+  }
+}
+
+.Testnet-warn {
```

Comment:
These are styles for separate components, and should be in their own respective .scss files.